tests/testing/make_sdists.py

```diff
@@ -6,12 +6,13 @@
 def random_string():
     """return a short suffix that shouldn't collide with any subsequent calls"""
     import os
-    import base64
+    import string
+    import random
 
-    return '.'.join((
-        str(os.getpid()),
-        base64.urlsafe_b64encode(os.urandom(3)),
-    ))
+    return '{}.{}'.format(
+        os.getpid(),
+        ''.join(random.choice(string.ascii_lowercase) for x in range(8)),
```

Comment:
```
>>> os.urandom(3).decode('us-ascii')                                                              
Traceback (most recent call last):                                                                
  File "<stdin>", line 1, in <module>                                                             
UnicodeDecodeError: 'ascii' codec can't decode byte 0x93 in position 0: ordinal not in range(128)
```

You could decode as latin1 but that seems derpy, what's the point of using base64 to get a random string?